frontend/packages/ceph-storage-plugin/src/components/independent-mode/fileUpload.scss

```diff
@@ -0,0 +1,22 @@
+.upload-component {
+  display: inline-flex;
+}
+
+.input-btn {
+  display: inline-flex;
+}
+
+.custom-input-btn {
+  flex-grow: 10;
+}
+
+.upload-btn__input {
+  position: absolute;
+  opacity: 0;
+  border: none;
+  max-width: 4em;
+}
+
+.upload-component__textInput {
+    color: var(--pf-global--Color--dark-200);
+}
```

Comment:
prefix with plugin package name

Reply:
I chose not to because this is not a component dedicated to a plugin rather a generic component. Which should actually go in the console/shared dir but for now.